jobserv/git_poller.py

```diff
@@ -308,19 +311,27 @@ def _trigger(entry: PollerEntry, trigger_name: str,
     }
     p = urlparse(change_params['GIT_URL'])
     url = p.scheme + '://' + p.netloc
+    commit_reason = ''
```

Comment:
The reason I'd mentioned you'd need to change the `_*log` functions to return None is because each one returns the the one line "subject" of the commit. These ci skip messages can occur anywhere in the commit subject and/or body. Here's some documentation: https://circleci.com/docs/2.0/skip-build/

Reply:
Thank you Andy for this, I believe I've now approached your concerns. Please resolve or clarify if I've misunderstood.